client/src/App.js

```diff
@@ -64,7 +64,10 @@ const App = props => {
   const getRefreshrs = () => {
     axios({
       method: 'get',
-      url: `http://localhost:9000/teachers/114/refreshrs`,
+      url: `http://localhost:9000/teachers/${user_id}/refreshrs`,
```

Comment:
Seems like logging in with this will throw an error (at least on local host):
`The server does not support SSL connections`
Small thing though